internal/files/shcd-schema.json

```diff
@@ -1,87 +1,82 @@
 {
```

Comment:
This doesn't seem to match the paddle schema from canu: https://github.com/Cray-HPE/canu/blob/develop/network_modeling/schema/paddle-schema.json

Reply:
fixed

Reply:
I still see some differences between the two.  Can you just copy the file over directly so they match?  Or do a git submodule to keep it in sync, or whatever method works.

Reply:
fixed in canu project and they match 100% now.